python/examples/advanced/Constraints_Suite.ipynb

```diff
@@ -0,0 +1,3619 @@
+       "        }\n",
+       "        handleSearch();\n",
+       "      });\n",
+       "\n",
+       "      $unknown.addEventListener(&quot;change&quot;, (event) =&gt; {\n",
+       "        const currentCheckbox = $(event.currentTarget);\n",
+       "\n",
+       "        if (event.currentTarget.checked) {\n",
+       "          $(&quot;.form-check-input&quot;).prop( &quot;checked&quot;, true );\n",
+       "          activeTypes[&quot;passed&quot;] = true;\n",
+       "          activeTypes[&quot;failed&quot;] = true;\n",
+       "          checkedBoxes();\n",
+       "        } else {\n",
+       "          $(&quot;.form-check-input&quot;).prop( &quot;checked&quot;, false );\n",
+       "          activeTypes[&quot;passed&quot;] = false;\n",
+       "          activeTypes[&quot;failed&quot;] = false;\n",
+       "        }\n",
+       "        handleSearch();\n",
+       "      });\n",
+       "    }\n",
+       "\n",
+       "    function checkedBoxes() {\n",
+       "      const $boxes = $(&#x27;input[name=checkbox]:checked&#x27;);\n",
+       "      const $notifCircleContainer = $(&quot;.notif-circle-container&quot;)\n",
+       "\n",
+       "      if ($boxes.length) {\n",
+       "        $notifCircleContainer.removeClass(&quot;d-none&quot;)\n",
+       "      }\n",
+       "    }\n",
+       "\n",
+       "    function openFilter() {\n",
+       "      const $filterOptions = $(&quot;.dropdown-container&quot;);\n",
+       "      const $notifCircleContainer = $(&quot;.notif-circle-container&quot;)\n",
+       "      const filterClass = $filterOptions.attr(&quot;class&quot;);\n",
+       "\n",
+       "      if (filterClass.indexOf(&quot;d-none&quot;) &gt; 0) {\n",
+       "        $notifCircleContainer.addClass(&quot;d-none&quot;)\n",
+       "        $filterOptions.removeClass(&quot;d-none&quot;);\n",
+       "        $(&quot;.filter-icon&quot;).addClass(&quot;d-none&quot;)\n",
+       "        $(&quot;.close-filter-icon&quot;).removeClass(&quot;d-none&quot;)\n",
+       "      } else {\n",
+       "        $filterOptions.addClass(&quot;d-none&quot;);\n",
+       "        $(&quot;.close-filter-icon&quot;).addClass(&quot;d-none&quot;)\n",
+       "        $(&quot;.filter-icon&quot;).removeClass(&quot;d-none&quot;)\n",
+       "        checkedBoxes()\n",
+       "      }\n",
+       "    }\n",
+       "\n",
+       "    // Invoke functions -- keep in mind invokation order\n",
+       "    registerHandlebarHelperFunctions();\n",
+       "    initHandlebarsTemplate();\n",
+       "    initWebsiteScripts();\n",
+       "  &lt;/script&gt;\n",
+       "&lt;/html&gt;\n",
+       "\" width=100% height=300\n",
+       "        frameBorder=0></iframe>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "builder = ConstraintsBuilder(dataset_profile_view=profile_view)\n",
```

Comment:
Running in collab I got an error:
`Unable to load pybars; install pybars3 to load profile directly from the current session `

Do we need to make sure that's installed at the top?

Reply:
Yes, installing whylogs[viz] will install pybars
I forgot to add it, thanks!